R/QueueDataWarehouse.R

```diff
@@ -71,7 +72,7 @@
 QueueDataWarehouse <- function(reportsuite.id, date.from, date.to, metrics, elements,
                                date.granularity='day', segment.id='', data.current=TRUE,
                                expedite=FALSE, interval.seconds=5, max.attempts=120,
-                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp) {
+                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp, format='csv') {
```

Comment:
One more comment...it doesn't appear that you are paginating the results, which is fine with me (I still think interactively returning a DW query to an R session is weird!), but we should put a note in the documentation. Something to the effect of 'The Adobe Analytics API paginates results at 20MB; pagination has not been implemented in QueueDataWarehouse. Should you need to return large amounts of data from a request, you should choose enqueueOnly = TRUE and send the results to an FTP'